### DISCUSSION TITLE
Additional insights from web3 reward emission research

### DISCUSSION THREAD

**Discussion by robertk16** [akash-network/community#240] [Show and tell]
We are currently undertaking a large-scale research effort to create better frameworks for token mechanics, especially for web3 protocols in the hardware resource provisioning sector. 

We have recently published part one, where we have compared emission schedules of various web3 networks, including of course Akash. You can find the report [here](https://mirror.xyz/1kx.eth/YWTZJO853QePwPUPka1aG4HXDT6hvqKnunG7fDe06Yg). 

However, we wanted to share some additional data with the Akash community that is not in the report. 

First off, we have classified Akash’s token rewards to be in the category of KPI-driven and decaying emissions. KPI-driven because the rewards paid depend on the stake rate and lock-up times. Over time the rewards decay and eventually approach zero, such that the total supply of AKT is limited.

Within that category, there are currently also the following projects we analyzed:
- Filecoin
- NYM

In comparison to the other projects in this category, Akash started higher but corresponding to the decrease of emissions over the past year, the cumulative rewards (relative to the maximum token supply) is ~28% to date which is slightly above the average of all Web3 infrastructure networks we analyzed.

The below chart shows the average (blue line) and the interquartile range (blue shaded) of the monthly token rewards of all Web3 infrastructure networks together with the projects of the KPI-driven and decaying emissions category highlighted:

![tkn_rel_to_maxsupply_all_Akash1](https://github.com/akash-network/community/assets/42434858/21de83bc-e1c6-4afe-b528-9b1aeb566359)

Next, we looked into the dollar value of those emissions, to compare how validators and stakers are actually getting compensated:

![tkn_usd_all_Akash1](https://github.com/akash-network/community/assets/42434858/a700767c-106e-4964-9d33-45aeb100ec04)

For most Web3 networks we looked at, the dollar-values of token emissions followed a quite different trajectory than the token rewards as prices were impacted by the 2021 bull market. Akash is one of the few exceptions where this is not the case, despite being around for the full bull market as well.

However, for Akash, it might be more important to compare it to the networks providing similar services versus projects with similar token emissions. So here we go. 

![tkn_rel_to_maxsupply_all_Akash6](https://github.com/akash-network/community/assets/42434858/2985decc-667d-49f0-bda0-cca5c7ff6b0c)

![tkn_usd_all_Akash6](https://github.com/akash-network/community/assets/42434858/c1f06d16-4a6d-44c4-b377-7842f4bc5b3b)

The values for render are the panned emissions based on [RNP001](https://github.com/rndr-network/RNPs/blob/main/Approved%20and%20on%20the%20Roadmap/rnp-001-bme.md), hence not started yet. More information and data are available in [this repo](https://github.com/1kx-network/web3infrastructure_tokenomics) which also contains the paper with further details on the published report. 

We hope this information yields some insightful information for the Akash community and are open to your feedback. What additional data would you want to see / aspects we should dive into?

**Comment by Danikovvv** [comment]
good
